docs/INTEGRATION_TUTORIAL.md

```diff
@@ -135,9 +202,33 @@ jobs:
 **Important Notes:**
 - The `fetch-depth: 0` fetches full history; you can use a specific depth (e.g., `fetch-depth: 50`) matching the `-n` flag used in `report` or `audit` commands
 - The `contents: write` permission is needed to push measurement data
+- **Git identity configuration is required** because git-perf stores measurements as git-notes (git commits)
 - The `git perf measure` command automatically times the execution of the supplied command
 - Push is unconditional to ensure measurements are always saved
 
+### Verification
+
+After pushing your workflow, verify it runs successfully:
+
+```bash
+# Push the workflow file
+git add .github/workflows/performance-tracking.yml
+git commit -m "ci: add performance tracking workflow"
+git push
+
+# Check the workflow status
+gh run list --workflow=performance-tracking.yml --limit 5
+
+# View the most recent run
+gh run view --log
+
+# Verify measurements were pushed
```

Comment:
Do verification with local execution of git perf pull and git perf report instead